src/lib-components/forms/DateRangePicker.vue

```diff
@@ -1,23 +1,33 @@
 <template>
-  <input
-    type="text"
-    class="shadow-sm focus:ring-blue-500 focus:border-blue-500 block w-full sm:text-sm border-gray-300 rounded-md"
-    placeholder="mm-dd-yyyy range"
-  />
+  <InputLabel
+    class="block"
+    :id="`${uuid}-label`"
+    :for="uuid"
+    :label="label"
+  ></InputLabel>
+  <BaseInput type="text" placeholder="mm-dd-yyyy range" :id="uuid"></BaseInput>
```

Comment:
My mind is getting a little jelly after the day, but 1. where is `BaseInput` coming from and 2. doesn't the base includ label and help?

Reply:
Good find.  Must have overlooked importing `BaseInput` when debugging with the dev site, since it makes use of `app.use(Trees);` and it became globally available.  It should be imported now and simplified to just using `BaseInput` with its label and help props.  Initially, it was composed with the label and help components to make initializing flatpickr easier with the uuid, but this reminded me that really the uuid should check for an $attrs.id before creating a unique.  So `uuid = (this.$attrs.id as string) || Uniques.CreateIdAttribute();` is being used which should play nicely with custom ID attributes that continue to work with aria tags.